src/differentElementArray.md

```diff
@@ -0,0 +1,52 @@
+# Different Element in Array
+This snip returns different elements between two arrays.
+
+## Result
+```
+ differentElemArray([1, 2, 3, 4, 5],[3, 4, 5, 6, 7]); //[1, 2]
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+* [Python](#python)
+* [Java](#java)
+
+### JavaScript
+```js
+function arrayDiff(array1, array2) {
```

Comment:
Check the function name and keep it the same as the example

Reply:
Updated the name